docs/source/usage.rst

```diff
@@ -255,6 +255,26 @@ This example creates a resolver that adds 10 the the given value.
     1000
 
 
+Custom resolvers support variadic argument lists in the form of a comma separated list of zero or more values.
```

Comment:
Please add a note saying this is from OmegaConf 1.3.1 (There are similar notes around the docs about deprecated things (which I need to clean up, duh)).

Reply:
Updated